frontend/src/Pages/Calculator/Calculator.js

```diff
@@ -1,138 +1,180 @@
 import React from "react";
 import { useState } from "react";
+import { Formik, Field, Form } from "formik";
 
 export function Calculator() {
-  const [age, setAge] = useState(21);
-  const [weight, setWeight] = useState(150);
-  const [height, setHeight] = useState(60);
+  const [bmr, setBmr] = useState(0);
+  const [tdee, setTdee] = useState(0);
+  const [calories, setCalories] = useState(0);
+  const [protein, setProtein] = useState(0);
+  const [fats, setFats] = useState(0);
+  const [carbs, setCarbs] = useState(0);
 
-  function handleAge(e) {
-    setAge(e.target.value);
-  }
+  function calculateBMR({age, weight, height, gender, alevel, goal, units}) {
+    let calculations = 0;
+    if (gender === 'male') {
+      if (units === 'imperial') calculations = 66.47 + (6.24 * weight) + (12.7 * height) - (6.75 * age);
+      else {
+        let kgWeight = weight * 0.45359237;
+        let cmHeight = height * 2.54;
+        calculations = 66.5 + (13.75 * kgWeight) + (5.003 * cmHeight) - (6.75 * age);
```

Comment:
Why are you using "66.5" here and "66.47" above?  And why are you using "5.003" here and "12.7" above?

Ideally, you could move this line into it's own function entirely:
- It seems like you're duplicating the logic four times here (and I think you made a  typo on line 27 - you wrote "655.1" instead of "65.51"!) 
- This is logic that can be well described and well encapsulated.

Reply:
Also - I think you can come up with a better name than "calculations" here.  This is "bmr" right?  Can we just call the variable "bmr"?